src/server.c

```diff
@@ -464,14 +468,14 @@ dictType zsetDictType = {
 };
 
 /* Db->dict, keys are sds strings, vals are Objects. */
-dictType dbDictType = {
-    dictSdsHash,          /* hash function */
-    NULL,                 /* key dup */
-    dictSdsKeyCompare,    /* key compare */
-    dictSdsDestructor,    /* key destructor */
-    dictObjectDestructor, /* val destructor */
-    dictResizeAllowed,    /* allow to resize */
-};
+dictType dbDictType = {dictSdsHash,          /* hash function */
+                       NULL,                 /* key dup */
+                       dictSdsKeyCompare,    /* key compare */
+                       NULL,                 /* key is embedded in the dictEntry and freed internally */
+                       dictObjectDestructor, /* val destructor */
+                       dictResizeAllowed,    /* allow to resize */
+                       .embedKey = dictSdsEmbedKey,
+                       .embedded_entry = 1};
 
```

Comment:
Was the previous version giving format errors.

Reply:
Yeah.
https://github.com/valkey-io/valkey/actions/runs/9704277218/job/26784193065

Reply:
I think I fixed it.